Source/EasyNetQ.Examples.PubSubWithAdvancedApi/EntryPoint.cs

```diff
@@ -2,23 +2,16 @@
 using EasyNetQ.Topology;
 
 var cts = new CancellationTokenSource();
+cts.CancelAfter(300000);
 Console.CancelKeyPress += (_, _) => cts.Cancel();
 
 using var bus = RabbitHutch.CreateBus(
     "host=localhost;publisherConfirms=True",
     x => x.EnableNewtonsoftJson()
         .EnableAlwaysNackWithoutRequeueConsumerErrorStrategy()
-        .EnableConsoleLogger()
 );
 
-var eventQueue = await bus.Advanced.QueueDeclareAsync(
-    "Events",
-    c => c.WithQueueType(QueueType.Quorum)
-        .WithOverflowType(OverflowType.RejectPublish),
-    cts.Token
-);
-
-using var eventsConsumer = bus.Advanced.Consume(eventQueue, (_, _, _) => { });
```

Comment:
If you remove consumer then this example is no more Pub**Sub**WithAdvancedApi.

Reply:
I have rolled it back. I used these changes to analyse memory allocations.